openfasoc/generators/ldo-gen/simulations/templates/ldoInst_ngspice.sp

```diff
@@ -52,30 +49,33 @@ R1 VREG VSS 3.6k
 
 *R0 VREG 0 R='TIME > 500u ? 720 : 3600' ;If want to do a transient simulation where I load is changing in the middle of a transient simulation. Example: Here it is changing from 720 ohm (2.5mA) to 3600 ohm (0.5 mA) at t = 1000us
 
-C1 VREG VSS 0.2n
+C1 VREG VSS 30n
```

Comment:
30nF?

Reply:
This will also be updated in the upcoming PR by @lulu9312

Reply:
Yes. I am planning to send a PR once this PR is merged so that we don't face any merge conflict issues.